gocd/templates/bash/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+eval $(/devinfra/scripts/regions/project_env_vars.py --region="${SENTRY_REGION}")
+/devinfra/scripts/k8s/k8stunnel
+/devinfra/scripts/k8s/k8s-deploy.py \
+  --type="statefulset" \
+  --label-selector="${LABEL_SELECTOR}" \
+  --image="us-central1-docker.pkg.dev/sentryio/peanutbutter/image:${GO_REVISION_PEANUTBUTTER_REPO}" \
+  --container-name="peanutbutter" \
```

Comment:
`us-central1-docker.pkg.dev/sentryio/peanutbutter` exists, but is thus far unpopulated.

Reply:
We should change the workflows in this repo to push the docker builds to the new location, this was also mentioned in the `ops` PR.

Reply:
This is done.